tests/acceptance/features/apiGraph/activities.feature

```diff
@@ -367,3 +367,172 @@ Feature: check activities
         }
       }
       """
+
+  @issue-9712 @issue-9744
+  Scenario: check move activity for a file
+    Given user "Alice" has uploaded file with content "ownCloud test text file 0" to "/textfile0.txt"
+    And user "Alice" has created folder "/New Folder"
+    And user "Alice" has moved file "textfile0.txt" to "New Folder/textfile0.txt"
+    When user "Alice" lists the activities of space "Personal" using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": ["value"],
+        "properties": {
+          "value": {
+            "type": "array",
+            "minItems": 3,
+            "maxItems": 3,
+            "uniqueItems": true,
+            "items": {
+              "oneOf": [
+                {
+                  "type": "object",
+                  "required": ["id","template","times"],
+                  "properties": {
+                    "template": {
+                      "type": "object",
+                      "required": ["message","variables"],
+                      "properties": {
+                        "message": {
+                          "const": "{user} added {resource} to {space}"
+                        },
+                        "variables": {
+                          "type": "object",
+                          "required": ["resource","space","user"],
+                          "properties": {
+                            "resource": {
+                              "type": "object",
+                              "required": ["id","name"],
+                              "properties": {
+                                "name": {
+                                  "const": "textfile0.txt"
+                                }
+                              }
+                            }
+                          }
+                        }
+                      }
```

Comment:
`message` should be enough for old activities
```suggestion
```

Reply:
wouldn't that be confusing to know which resource it is for?

Reply:
for a single item, we should get the activities of that item only as noted here https://github.com/owncloud/ocis/pull/9731#discussion_r1714759795
That we won't have to deal with multiple items. but the point is true for folder activities :+1: